src/main/java/io/supertokens/storage/mysql/HikariLoggingAppender.java

```diff
@@ -129,7 +129,7 @@ public void setName(String name) {
     @Override
     public void doAppend(ILoggingEvent event) throws LogbackException {
         if (event.getLevel() == Level.ERROR) {
-            Logging.error(start, event.getFormattedMessage(), false);
+            Logging.error(start, event.getFormattedMessage(), true);
```

Comment:
Do you know what kind of error logs are sent by Hikari and how often?

Reply:
Here are the logged errors I found in [HikariCP](https://github.com/brettwooldridge/HikariCP/search?q=error):

```
"Property {} does not exist on target {}"
"{} - JMX name ({}) is already registered."
"{} - Failed to execute{} connection test query ({})."
"Failed to load driver class {} from HikariConfig class classloader {}"
"{} - Error thrown while acquiring connection from data source"
```

They appear to be entirely configuration and connection errors, so I wouldn't expect them to appear often.